client/state/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import thunkMiddleware from 'redux-thunk';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import sharing from './sharing/reducer';
+import sites from './sites/reducer';
+import ui from './ui/reducer';
+
+/**
+ * Module variables
+ */
+const reducer = combineReducers( {
+	sharing,
+	sites,
+	ui
+} );
+
+export function createReduxStore() {
```

Comment:
And as for Server-Side rendering:
What sense does keeping an application state make for SSR? How do we match state of store with the url to serve a page with a different state?
I'm kind of confused by this, because I was quite certain SSR needs only to match the URL and will not need to mutate the store.